components/badge/badge.css

```diff
@@ -0,0 +1,99 @@
+@import url(/landing.css);
+@import url(/components/instalation/instalation.css);
+@import url(/components/avatar/avatar.css);
+
+
+.all_badges {
+    padding: 4rem;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+
+.status_badge {
+    border: 2px solid white;
+    position: relative;
+    border-radius: 50%;
+}
+
+.large_badge {
+    height: 1.5rem;
+    width: 1.5rem;
+    bottom: 0.2rem;
+    right: 3rem;
+    background-color: green;
+}
+
+.medium_badge {
+    height: 1rem;
+    width: 1rem;
+    bottom: 0.2rem;
+    right: 2.5rem;
+    border: 4px solid darkgrey;
+    background-color: grey;
+}
+
+.small_badge {
+    height: 0.7rem;
+    width: 0.7rem;
+    bottom: 0.1rem;
+    right: 1.5rem;
+    background-color: red;
+
+}
+
+/*    icon badge */
+
+.icon-badge_div {
+    display: flex;
+    flex-direction: row;
```

Comment:
You can remove flex-direction: row, as it is by default row.

Reply:
okk i will do that.